CONTRIBUTING.md

```diff
@@ -212,6 +212,8 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 Like all languages, Dream Maker has its quirks, some of them are beneficial to us, like these
 
 * In-To for loops: ```for(var/i = 1, i <= some_value, i++)``` is a fairly standard way to write an incremental for loop in most languages (especially those in the C family) however DM's ```for(var/i in 1 to some_value)``` syntax is oddly faster than its implementation of the former syntax; where possible it's advised to use DM's syntax. (Note, the ```to``` keyword is inclusive, so it automatically defaults to replacing ```<=```, if you want ```<``` then you should write it as ```1 to some_value-1```).
+HOWEVER, if either ```some_value``` or ```i``` changes within the body of the for (underneath the for(...) header) or If you are looping over a list AND changing the length of the list then you can NOT use this type of for loop!
```

Comment:
Some formatting mistakes:
```
underneath the ```for(...)``` header
```
missed the backtics

or *If* you are looping
should be
or if you are looping